km61/Drapak_Volodymyr/laboratory1.sql

```diff
@@ -1,2 +1,144 @@
 -- LABORATORY WORK 1
 -- BY Drapak_Volodymyr
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+оновлювати дані в таблицях
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Drapak
+IDENTIFIED BY qwerty123456;
+
+ALTER USER Drapak
+    GRANT ALTER ANY TABLE TO Drapak;
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Людина дивиться кіно.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE Human(
+    Name VARCHAR(30) NOT NULL,
+    Age NUMBER(3, 3) NOT NULL
+)
+
+CREATE TABLE Film(
+    Name VARCHAR(100) NOT NULL,
+    Human_name_fk VARCHAR(100) NOT NULL,
+    Genre VARCHAR(20) NOT NULL
+)
```

Comment:
Need to use structure `CREATE ...(...);`

Reply:
I used it in my primary code. You've mistaken with smth.